menu.html

```diff
@@ -9,9 +9,13 @@
   <li><a href="/requirements?{{ session_query }}">Requirements</a>
   <li><a href="/groups_classes?{{ session_query }}">Class Groups</a>
   <li><a href="/groups_students?{{session_query }}">Student Groups</a>
+  <li><a href="/rosters?{{session_query }}">Class Rosters</a>
+  <li><a href="/serving_rules?{{session_query }}">Serving Rules</a>
 
-  <h3>Error Check</h3>
+  <h3>Configure</h3>
   <li><a href="/error_check?{{ session_query }}">Error Check</a>
+  <li><a href="/init_rosters?{{ session_query }}">Initialize Rosters</a>
```

Comment:
Maybe change these 2 lines to buttons in the "automated" page linked on line 23?